samples/bluetooth/peripheral_uart/src/main.c

```diff
@@ -451,7 +451,7 @@ static void led_blink_thread(void)
 			bt_conn_auth_cb_register(&conn_auth_callbacks);
 		}
 
-		err = k_sem_take(&ble_init_ok, K_MSEC(100));
+		err = k_sem_take(&ble_init_ok, K_MSEC(1000));
```

Comment:
Bluetooth init over OpenAMP takes a while

This is temporary fix, I would like to get rid of this complex thread structure in peripheral_uart sample.